src/ovh-angular-mondial-relay/ovh-angular-mondial-relay.directive.js

```diff
@@ -212,9 +227,27 @@ angular.module("ovh-angular-mondial-relay")
                     this.loading.search = true;
                     this.ngModel = null;
                     this.foundRelays = [];
+                    var filt = filter;
+
+                    if (typeof filt === "undefined") {
```

Comment:
this cannot be replaced by `if (filt)` ?

Reply:
Does it change anything if it's `null`?